orbit/pkg/packaging/macos.go

```diff
@@ -316,9 +316,9 @@ func cpio(srcPath, dstPath string) error {
 	if err != nil {
 		return errors.Wrap(err, "wait gzip")
 	}
-	err = dst.Close()
+	err = dst.Sync()
```

Comment:
It doesn't look like `Sync` also includes what `Close` does, should we also close in this case?

Reply:
It doesn't show up in the short diff, but the `Close` is already deferred at the top of the function.